src/shared/stubs/Object.create.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2014-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule Object.create
+ */
+
+// https://people.mozilla.org/~jorendorff/es6-draft.html#sec-object.create
+
+'use strict';
+
+var nativeCreate = typeof Object.create === 'function' && Object.create;
```

Comment:
This seems superfluous to me at least. `Object.create` should be enough, a broken environment is not our problem I would think...